tracetools_launch/package.xml

```diff
@@ -16,6 +16,8 @@
   <depend>launch_ros</depend>
   <depend>tracetools_trace</depend>
 
+  <exec_depend>launch</exec_depend>
```

Comment:
When I try to run `rosdoc2` (https://github.com/ros-infrastructure/rosdoc2/commit/489e3a39e109387d7ef2af92adaff14bab685765) on `tracetools_launch`, I get:

```
$ rosdoc2 build --package-path src/ros2/ros2_tracing/tracetools_launch
Failed to parse ROS package manifest in 'src/ros2/ros2_tracing/tracetools_launch': Error(s) in package 'src/ros2/ros2_tracing/tracetools_launch/package.xml':
Error(s):
- The generic dependency on 'launch' is redundant with: exec_depend
```

Reply:
Yes, that's true.  In point of fact, this package probably should *not* have a `<depend>` tag on `launch`, and instead only have the `exec_depend`.  The same goes for `launch_ros`.

Reply:
And `tracetools_trace`?

Reply:
Apologies for the oversight! Fixed in [3bfd58a](https://github.com/ros2/ros2_tracing/pull/50/commits/3bfd58af514240ab65a6733b8e0b9351008ee34b)

I'm still facing some import warnings but the autodoc tool should succeed now. (the generated doc will be incomplete)

```
WARNING: autodoc: failed to import module 'tracetools_launch'; the following exception was raised:
cannot import name 'logging' from 'launch' (unknown location)
WARNING: autodoc: failed to import module 'action' from module 'tracetools_launch'; the following exception was raised:
cannot import name 'logging' from 'launch' (unknown location)
WARNING: autodoc: failed to import module 'actions' from module 'tracetools_launch'; the following exception was raised:
cannot import name 'logging' from 'launch' (unknown location)
WARNING: autodoc: failed to import module 'actions.ld_preload' from module 'tracetools_launch'; the following exception was raised:
cannot import name 'logging' from 'launch' (unknown location)
```

Reply:
Like you said, the generated docs are basically empty, but the tool does succeed using your branch.